pom.xml

```diff
@@ -72,7 +72,6 @@
         <workflow-support-plugin.version>3.7</workflow-support-plugin.version>
         <node.version>12.19.0</node.version>
         <npm.version>6.14.8</npm.version>
-        <frontend-version>1.10.0</frontend-version>
```

Comment:
this is overriding the parent and building breaks on my machine, node and npm could probably do with an update too but just wanting to compile for now